control/content/js/classes/analytics.js

```diff
@@ -1,18 +1,36 @@
 class Analytics {
   constructor() {}
   // Register plugin events for analytics
-  static registerEvent(title, key, description, silentNotification = true) {
-    buildfire.analytics.registerEvent(
-      {
-        title: title,
-        key: key,
-        description: description,
-      },
-      { silentNotification }
-    );
-  }
+  static registerEvent = (event, options, callback) => {
```

Comment:
The version of lint that is used in this project is failing because it is not supporting arrow functions, please update the lint or use standard functions
you can see the error when running **_npm run build_** command

Error message:
\js\classes\analytics.js
  4:24  error  Parsing error: Unexpected token =